kafkaesk/app.py

```diff
@@ -440,14 +450,20 @@ async def __aenter__(self) -> "Application":
         await self.initialize()
         return self
 
-    async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
+    async def __aexit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc: Optional[BaseException],
+        traceback: Optional[TracebackType],
+    ) -> None:
+        logger.warning("Stopping application...", exc_info=exc)
```

Comment:
Do we want log level warning when stopping? Maybe just info?

Or check `if exc is not None` before logging?